docs/catalog.md

```diff
@@ -370,6 +370,38 @@ Here is an example of creating a package binding and identifying the language of
   }
   ```
 
+### Synthesizes text input to spoken audio
+
+The `/whisk.system/watson/textToSpeech` action help synthesize text input of any supported language to produce audio output. The parameters are as follows:
+
+- `username`: The Watson API username.
+- `password`: The Watson API password.
+- `payload`: The text to be synthesized.
+- `voice`: The voice to be used for synthesis.
+- `accept`: The requested MIME type of the audio.
+- `encoding`: The encoding of the speech binary data.
+
+Here is an example of creating a package binding and synthesize the audio of some input text.
```

Comment:
Here is an example of creating a package binding and synthesize the audio of some input text. ->
The following example creates a Watson text to speech package binding to synthesize input text to audio.